spec/neo4j/http/cypher_client_spec.rb

```diff
@@ -83,5 +83,38 @@
         stubs.verify_stubbed_calls
       end
     end
+
+    describe "execute_batch_cypher" do
+      it "executes all statments" do
```

Comment:
Can you add a test for if statement2 fails? Will it continue on and do the rest of the statements?

Reply:
Added.